src/modules/esl-carousel/renderers/esl-carousel.css-only.renderer.less

```diff
@@ -1,60 +1,67 @@
-/** @deprecated mode is under development at that moment */
 .esl-carousel-css-renderer {
   @SLIDE_CAROUSEL_ANIMATION: 2s linear;
 
+  [data-slides-area] {
+    display: block;
+    position: relative;
+    width: 100%;
+    --offset: 0;
+    transform: translate3d(var(--offsetArea, 0), 0, 0);
+  }
+
+  &[animating] [data-slides-area] {
```

Comment:
In case this option conflicts with the main animation but is created for touch/drug support we need to consider separate marker for that one but not for main animation.